docs/apis/subsystems/check/index.md

```diff
@@ -225,6 +225,20 @@ https://github.com/moodle/moodle/blob/master/lib/classes/check/access/riskxss_re
 
 Some checks are by their nature asynchronous. For instance having moodle send an email to itself and then having it processed by the inbound mail handler to make it's properly configured (see [MDL-48800](https://tracker.moodle.org/browse/MDL-48800)). In cases like these please make sure the age or staleness of the check is shown in the summary, and you should also consider turning the result status into a warning if the result is too old. If appropriate make the threshold a configurable admin setting.
 
+## Reusing checks inside admin setting pages
+
+Quite often you have a plugin that includes some sort of status check in the admin settings page where you configure the plugin. Eg you can add the api keys for a service and it shows you right in the settings page if the connection settings work.
```

Comment:
```suggestion
Quite often you have a plugin that includes some sort of status check in the admin settings page where you configure the plugin. For example you can add the api keys for a service and it shows you right in the settings page if the connection settings work.
```

We try to avoid `eg` and `ie` because they don't translate well to non-EFL speakers.

See:
- https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg
- https://developers.google.com/style/abbreviations#:~:text=Abbreviations%20not%20to%20use,-Prefer%20English%20terms&text=Don't%20use%20i.e.%20or,ymmv%2C%20RTFM%2C%20or%20others.